pkgs/data/misc/dns-root-data/default.nix

```diff
@@ -4,7 +4,7 @@ let
 
   rootHints = fetchurl {
     url = "http://www.internic.net/domain/named.root";
```

Comment:
Changes of a root server IP happened a couple times during the last year.  [Next change is scheduled for October.](http://root-servers.org/news.html)